contracts/argent_account.cairo

```diff
@@ -47,6 +48,44 @@ mod ArgentAccount {
         interface_id == ERC165_OLD_ACCOUNT_INTERFACE_ID
     }
 
+    // ERC1271
+    #[view]
+    fn isValidSignature(ref signatures: Array::<felt>, message_hash: felt) -> bool {
```

Comment:
As most of this code is argent specific, I'd move this logic in another file

Reply:
it's fine having argent specific code in the ArgentAccount contract

Reply:
I'd wait for the full contract to be done before thinking about factoring

Reply:
Agree with that.